app/assets/images/icons/folder.svg

```diff
@@ -0,0 +1,3 @@
+<svg width="15" height="13" viewBox="0 0 15 13" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
This looks to be the folder icon from Material Icons -- can we use that instead?